charts/nexus-iq/templates/persistentVolumeClaim.yaml

```diff
@@ -27,6 +28,8 @@ spec:
   storageClassName: "{{ .Values.persistence.storageClass }}"
 {{- end }}
 {{- end }}
+{{- end }}
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingLogClaim) }}
```

Comment:
the log pvc was being controlled by existence of `existingClaim` (top of this file) instead of independently being connrolled by its own parameter.

Reply:
weird seeing an added end & the `if` coming afterwards!